client/signup/steps/design-picker/let-us-choose.tsx

```diff
@@ -0,0 +1,67 @@
+import styled from '@emotion/styled';
+import { Button } from '@wordpress/components';
+import { useTranslate } from 'i18n-calypso';
+import { recordTracksEvent } from 'calypso/lib/analytics/tracks';
+import type { Design } from '@automattic/design-picker';
+
+interface Props {
+	flowName: string;
+	designs: Design[];
+	onSelect: ( design: Design ) => void;
+}
+
+const LetUsChooseContainer = styled.div`
+	color: var( --color-neutral-70 );
+	font-size: 14px;
+	margin-bottom: 40px;
+
+	@media ( min-width: 600px ) {
+		margin-top: 40px;
+	}
+`;
+
+const LetUsChooseButton = styled( Button )`
+	border-radius: 4px;
+	height: 40px;
+	font-weight: 500;
+	margin-top: 20px;
+	text-align: center;
+`;
+
+const LetUsChoose = ( { flowName, designs, onSelect }: Props ) => {
+	const translate = useTranslate();
+
+	const defaultDesign = designs.find( ( design ) =>
+		design.features.includes( 'difm-lite-default' )
+	);
+
+	if ( ! defaultDesign ) {
+		return null;
+	}
+
+	function onClick() {
+		recordTracksEvent( 'calypso_signup_let_us_choose_click', {
+			flow: flowName,
+		} );
+		// @ts-expect-error: TS complains that defaultDesign could
+		// be undefined. But we already return early if it is so.
+		onSelect( defaultDesign, {
+			isLetUsChooseSelected: true,
+		} );
+	}
+
+	return (
+		<LetUsChooseContainer>
+			<div>
+				{ translate(
+					"Can't decide? " + 'No problem, our experts can choose the perfect design for your site!'
```

Comment:
Do we need to concatenate this string?

Reply:
The only reason is to make it easier for translators: https://github.com/Automattic/wp-calypso/blob/trunk/packages/i18n-calypso/README.md#L44

Reply:
I'm not sure how the `+` makes it easier for translators 🤔 The concat operator is allowed for code styling reasons, the long string still appears as a single value to translators. For example, [this long string that uses concatenation in the code](https://github.com/Automattic/wp-calypso/blob/294238af02e1f3e67205c27af040c3afcd94b2d7/client/blocks/edit-gravatar/index.jsx#L214) appears as [a single continuous string in glotpress](https://translate.wordpress.com/projects/wpcom/es/default/?filters%5Bstatus%5D=either&filters%5Boriginal_id%5D=231250&filters%5Btranslation_id%5D=9624258).

I tried removing the `+` and Prettier was happy with the resulting line length, so I think it's more idiomatic to not concat in this instance.